blueprint-manager/src/main/java/com/sequenceiq/cloudbreak/blueprint/template/BlueprintTemplateModelContextBuilder.java

```diff
@@ -82,12 +82,12 @@ private BlueprintTemplateModelContextBuilder withFileSystemConfigurationView(Fil
     }
 
     public BlueprintTemplateModelContextBuilder withLdap(LdapConfig ldapConfig) {
-        this.ldap = Optional.ofNullable(ldapConfig == null ? null : new LdapView(ldapConfig));
+        ldap = Optional.ofNullable(ldapConfig == null ? null : new LdapView(ldapConfig));
```

Comment:
I suggest to refactor method to wait for `LdapView` as input parameter

Reply:
I don't think so. LdapView used inside in the module

Reply:
I messed because `public BlueprintTemplateModelContextBuilder withGateway(GatewayView gatewayConfig)` where `GatewayView` comes outside.

Reply:
thats also a problem